config/crd/kustomization.yaml

```diff
@@ -1,39 +1,5 @@
+# Generated by scripts/generate-crd-kustomize.sh. DO NOT EDIT.
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-
-# This kustomization.yaml is not intended to be run by itself,
-# since it depends on service name and namespace that are out of this kustomize package.
-# It should be run by config/default
 resources:
-- bases/gateway-operator.konghq.com_controlplanes.yaml
-- bases/gateway-operator.konghq.com_dataplanes.yaml
-- bases/gateway-operator.konghq.com_gatewayconfigurations.yaml
-- bases/gateway-operator.konghq.com_dataplanemetricsextensions.yaml
-- bases/gateway-operator.konghq.com_kongplugininstallations.yaml
-- bases/gateway-operator.konghq.com_konnectextensions.yaml
-#+kubebuilder:scaffold:crdkustomizeresource
-
-# patches:
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix
-# and ensure they are enclosed under patches key above.
-# patches here are for enabling the conversion webhook for each CRD
-#- path: patches/webhook_in_controlplanes.yaml
-#- path: patches/webhook_in_dataplanes.yaml
-#- path: patches/webhook_in_gatewayconfigurations.yaml
-#- patches/webhook_in_dataplanes.yaml
-#- patches/webhook_in_aigateways.yaml
-#+kubebuilder:scaffold:crdkustomizewebhookpatch
-
-# [CERTMANAGER] To enable cert-manager, uncomment all the sections with [CERTMANAGER] prefix
-# and ensure they are enclosed under patches key above.
-# patches here are for enabling the CA injection for each CRD
-#- path: patches/cainjection_in_controlplanes.yaml
-#- path: patches/cainjection_in_dataplanes.yaml
-#- path: patches/cainjection_in_gatewayconfigurations.yaml
-#- patches/cainjection_in_dataplanes.yaml
-#- patches/cainjection_in_aigateways.yaml
-#+kubebuilder:scaffold:crdkustomizecainjectionpatch
-
-# the following config is for teaching kustomize how to do kustomization for CRDs.
-configurations:
-- kustomizeconfig.yaml
+  - https://github.com/kong/kubernetes-configuration/config/crd/gateway-operator?ref=v1.1.1-0.20250217181409-44e5ddce290d # Version is auto-updated by the generating script.
```

Comment:
Since we are using the unreleased version, I would add a logic to the generate script to fix this